samples/snippets/src/main/java/pubsub/PublishWithGrpcCompressionExample.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pubsub;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.pubsub.v1.Publisher;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.TopicName;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+public class PublishWithGrpcCompressionExample {
+  public static void main(String... args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    // Choose an existing topic.
+    String topicId = "your-topic-id";
+
+    publishWithGrpcCompressionExample(projectId, topicId);
+  }
+
+  public static void publishWithGrpcCompressionExample(String projectId, String topicId)
+      throws IOException, ExecutionException, InterruptedException {
+    TopicName topicName = TopicName.of(projectId, topicId);
+
+    // Create a publisher and set enable compression to true.
+    Publisher publisher = null;
+    try {
+      // Enable compression and configure the compression threshold to 10 bytes (default to 240 B).
```

Comment:
Nit: Should we replace "bytes" to "B" just to be consistent?

Reply:
Thank you @rajanya-google. I actually wanted to show the unit spelled out followed by the the unit shorthand. Ack.